### DISCUSSION TITLE
No root node in result

### DISCUSSION THREAD

**Discussion by kkestner1** [NaturalIntelligence/fast-xml-parser#508] [General]
Is there any way to not have the root node in the generated result?

Example:
`< root >
<tag1>value1</tag1>
<tag2>value2</tag2>
< /root >`

becoming:
`
{
"tag1": "value1";
"tag2":"value2";
}
`

Thanks!

**Comment by amitguptagwl** [comment]
With the current implementation, No!. You can probably assign the value or result like `result = result.root`

**Reply by kkestner1** [reply]
Thank you!   Couple more questions....

Using the "isArray" function,  if two items at the same level have the same tag name, does it auto combine the two items into one array, or does it just make each individual item its own array?   Biggest issue I'm having right now tryng some other parsers...

I noted that in the implementations of transformTagName and tagValueProcesser, that you have to provide the entire jpath.
Would be so much easier if I only had to provide just the parent tag name and the item's tag name, vs the entire path, since what I'm trying to rearchitect is fairly complex with some items appearing at various levels.   Any chance that could be implemented in a future release?

**Comment by amitguptagwl** [comment]
Well! I'm not sure if you have used the library yet. Hence, you have these questions. So please try it once.
`transformTagName` is called with tagName. Probably we need to improve it so tag path is also passed.
`tagValueProcesser` is called with complete path of the tag. Now it's up to you how you use it. What you're taking about is I think stopNode. But it already supports pattern like "*.b". So I'm not sure what do you want to ask.
